docs/overlays/custom-overlays.md

```diff
@@ -1,50 +1,57 @@
 # Custom Overlays
 
 Because overlays are regular React components, it's straightforward to create
-resuable overlays that others can include into their project. Overlays typically
-take at least the following viewport props.
+resuable overlays that others can include into their project. Overlays can access
+the current viewport through the React context:
```

Comment:
Should we add a quick link to https://facebook.github.io/react/docs/context.html for people who might be unfamiliar with `context`?